Async/README.md

```diff
@@ -86,20 +86,26 @@ Status: success
 
 We can see that the Status is `success` which means the call finished properly. And you can see the time it started and completed.
 
-But how do we check the logs to debug and ensure things ran properly? There's a `log` endpoint to allow you to check this and you can 
+But how do we check the logs to debug and ensure things ran properly? There's a `log` endpoint to allow you to check this and you can
 access it via the CLI with:
 
 ![user input](../images/userinput.png)
 >```sh
->fn get logs myapp CALL_ID
+>fn get logs asyncapp CALL_ID
 >```
 
 For this function, the logs will contain:
 
 ```json
-{"message":"Hello World"}
+{"body":"{\"message\":\"Hello World\"}\n","content_type":"","protocol":{"status_code":200}}
 ```
 
+A JSON response is returned with three fields.
+
+* **body:** The output from the function. In this case an escaped version of the JSON response message: `{"message":"Hello World"}`
+* **content_type:** The content type of the response message. Empty in this case which defaults to `application/json`.
```

Comment:
The  empty content type is a problem in the Go FDK.  The boilerplate code doesn't specify it so it should either not be in the response or default to something generic like application/text as we are doing when `fn call` is invoked with no content type specified. The fact that the FDK is automatically logging the result (with user data) to stderr is also a problem I'll look into. 

For now I'd comment out the content_type discussion until we can get it cleared up.

Reply:
Hmm, might not be the FDK logging the result.  Might be the Fn server.

Reply:
I will just take out that bullet for now.